Component/button/button.css

```diff
@@ -1,5 +1,3 @@
-@import url(/CSS/docs.css);
-
 
 .btn{
```

Comment:
just found this class undeclared , this could be considered during refactoring later

Reply:
didnt get you